client/views/join_channel.tpl

```diff
@@ -0,0 +1,6 @@
+<form id="join_channel_{{id}}" class="join_form" method="post" action="" style="display: none;">
+	<input type="text" class="input" name="channel" placeholder="#channel" pattern="[^\s]+" maxlength="200" title="Should be a valid channel name">
+	<input type="password" class="input" name="key" placeholder="channel key" pattern="[^\s]+" title="Should be a valid channel key" maxlength="200" >
+	<button type="submit" class="btn joinchan:submit" data-id="{{id}}">Join</button>
```

Comment:
The button text does not have the correct color. I think we need to remove `#sidebar button` at https://github.com/thelounge/lounge/blob/master/client/css/style.css#L457 (just the line, not the whole block) but I just did a super quick test, nothing too involved.